src/bzflag/SceneRenderer.cxx

```diff
@@ -317,6 +321,15 @@ void SceneRenderer::setRebuildTanks()
 }
 
 
+void SceneRenderer::bzdbCallback(const std::string& name, void *)
+{
+    if(name == "radarsize")
```

Comment:
There should be a space between the `if` and open parenthesis

Reply:
Sorry about that.